bin/allowlist

```diff
@@ -0,0 +1,5 @@
+telemetry/core
+org-mozilla-fenix/.*
+org-mozilla-reference-browser/.*
+activity-stream/.*
+eng-workflow/bmobugs
```

Comment:
```suggestion
eng-workflow/bmobugs
mobile/activation
```

This new ping was just merged in https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/331 and I'd like to start testing it in the gcp pipeline tomorrow.